keyboards/archetype/minervalx/config.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2024 Archetype Made, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+
+#define WS2812_PIO_USE_PIO1
+
+#define RGBLIGHT_DEFAULT_MODE RGBLIGHT_MODE_RAINBOW_SWIRL
+
```

Comment:
```suggestion
```

Reply:
Aren't these lines needed for the WS2812 Driver?

Reply:
Are you using the WS2812 driver?

Reply:
You're right! No need for the driver as we're only using Through-Hole UV LEDs.